src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProject.cs

```diff
@@ -85,6 +85,11 @@ public override async Task<string> GetAssetsFilePathOrNullAsync()
             return await GetAssetsFilePathAsync(shouldThrow: false);
         }
 
+        public override Task AddFileToProjectAsync(string filePath)
```

Comment:
Maybe leave a comment that this is unnecessary/why it's not implemented in the SDK-based world.